Documentation/LSP Extensions.md

```diff
@@ -376,3 +376,47 @@ Users should not need to rely on this request. The index should always be update
 ```ts
 export interface TriggerReindexParams {}
 ```
+
+## `sourcekit-lsp/peekDocuments`
```

Comment:
All the other LSP extensions are named according to “normal” LSP requests, like `workspace/tests`. Let’s also name this as `workspace/peekDocuments`.